Backpack-SwiftUI/SearchInputSummary/Classes/BPKSearchInputSummary.swift

```diff
@@ -35,22 +36,33 @@ public struct BPKSearchInputSummary: View {
     @Binding private var text: String
     @FocusState private var focused: Bool
     private let placeholder: String
-    private let inputPrefix: InputPrefix
+    private let inputPrefix: InputPrefix?
     private var state: State = .default
     
-    /// Creates a `BPKSearchInputSummary`.
-    ///
-    /// - Parameters:
-    ///   - placeholder: The placeholder text to display when the text field is empty.
-    ///   - inputPrefix: The prefix which would be displayed on the left of text input
-    ///   - text: The text to display in the text field.
+    private let readOnly: Bool
+    private let clearAction: ClearAction?
```

Comment:
per figma, this should not be optional

Reply:
Before it was optional, I need to double check it with Adam

Reply:
yes, we can make it non-optional